docs/admin/k8s-deployment.md

```diff
@@ -1,172 +1,404 @@
-# How to Manually Deploy bpfd on Kubernetes
+# How to Deploy bpfd on Kubernetes
 
-## Pre-requsites
+One of the key use cases for bpfd is to support the deployment of bpf programs
+in kubernetes clusters.  The simplest way to do this is to use the
+bpfd-operator.  In this document, we provide an overview of using the
+bpfd-operator as well as the manual steps required to use bpfd with kubernetes,
+in general, and OpenShift, in particular.
 
-- [kind](https://kind.sigs.k8s.io/docs/user/quick-start/)
-  - [kind](https://kind.sigs.k8s.io/docs/user/quick-start/) can be used for a local development cluster.
-    This is optional and `bpfd` can be deployed on any kubernetes cluster.
-  - If using a `kind` cluster, the following command will create the cluster:
-    ```bash
-    kind create cluster
-    ```
+## bpfd-operator Overview
 
-- [cert-manager](https://cert-manager.io/)
-  - [cert-manager](https://cert-manager.io/) is used for certificate management.
-  - Full install instructions for cert-manager can be found [here](https://cert-manager.io/docs/installation/).
-  - To use the default static install instructions simply run:
-    ```bash
-    kubectl apply -f https://github.com/cert-manager/cert-manager/releases/download/v1.9.1/cert-manager.yaml
-    ```
-    This will create three pods in the `cert-manager` namespace:
-    ```bash
-    kubectl get pods -n cert-manager
-    NAME                                         READY   STATUS    RESTARTS   AGE
-    cert-manager-5dd59d9d9b-bbqbx                1/1     Running   0          22h
-    cert-manager-cainjector-8696fc9f89-rxtm7     1/1     Running   0          22h
-    cert-manager-webhook-7d4b5b8c56-gt6xb        1/1     Running   0          22h
-    ```
+The Bpfd-Operator can be used to deploy and manage bpfd within a kubernetes cluster.
 
-## Bpfd Component install
+![bpfd on K8s](./bpfd-on-k8s.jpg)
```

Comment:
Drawing needs to be updated to remove `E` from EbpfProgramConfig and EbpfProgram.